### PR TITLE
Issue #3110310 by wengerk, rattusrattus, sahaj, interdruper, gido: Blocks rendered via bamboo_render_block do not use the block theme hook

### PR DESCRIPTION
**Pull request by WengerK**
### 💬 Describe the pull request/code changes
_A clear and concise description of what the pull request is about._
Issue #3110310 by wengerk, rattusrattus, sahaj, interdruper, gido: Blocks rendered via bamboo_render_block do not use the block theme hook

### 🎟️ Drupal.org
_This pull request is related to the following issue(s):_
https://www.drupal.org/project/bamboo_twig/issues/3110310

### 📝 Checklist
- [x] Update the "Unreleased" section of the `CHANGELOG.md` with [chan](https://github.com/geut/chan/tree/main/packages/chan).  _(if applicable)_
- [x] The PR is covered by some tests. _(if applicable)_

### CODE REVIEW

File: tests/src/Kernel/Render/ContentBlockTest.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  public function testRenderContentBlock() {
+    $block = $this->createBlockContent();
+    $this->placeBlock('block_content:' . $block->uuid());
+
+    // Ensure {{ bamboo_render_block('block_content:ca1f2401-1') }}.
+    $renderer = $this->renderExtension->renderBlock('block_content:' . $block->uuid(), [], FALSE);
+    $this->assertArrayHasKey('#block_content', $renderer);
+    $this->assertInstanceOf(BlockContent::class, $renderer['#block_content']);
+
+    // Ensure {{ bamboo_render_block('block_content:ca1f2401-1', [], FALSE) }}.
```
Comment (gido) [inline tests/src/Kernel/Render/ContentBlockTest.php]:
```suggestion
    // Ensure {{ bamboo_render_block('block_content:ca1f2401-1', [], TRUE) }}.
```
?

File: tests/src/Kernel/Loader/EntityBlockTest.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    $this->createBlockContentType('basic');
+
+    /** @var \Drupal\bamboo_twig_loader\TwigExtension\Loader $loaderExtension */
+    $this->loaderExtension = $this->container->get('bamboo_twig_loader.twig.loader');
+  }
+
+  /**
+   * @covers ::loadEntity
+   *
+   * Cover the usage of {{ bamboo_render_entity('block', 'stark_branding') }}.
```
Comment (WengerK) [inline tests/src/Kernel/Loader/EntityBlockTest.php]:
```suggestion
   * Cover the usage of {{ bamboo_load_entity('block', 'stark_branding') }}.
```

File: tests/src/Kernel/Loader/EntityBlockTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+   */
+  public function testLoaderBlockEntity() {
+    $entity = $this->loaderExtension->loadEntity('block', 'test_block');
+    $this->assertInstanceOf(Block::class, $entity);
+  }
+
+  /**
+   * @covers ::loadEntity
+   *
+   * Cover the usage of {{ bamboo_render_entity('block_content', 1) }}.
```
Comment (WengerK) [inline tests/src/Kernel/Loader/EntityBlockTest.php]:
```suggestion
   * Cover the usage of {{ bamboo_load_entity('block_content', 1) }}.
```

File: tests/src/Kernel/Render/ContentBlockTest.php [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    /** @var \Drupal\bamboo_twig_loader\TwigExtension\Render $renderExtension */
+    $this->renderExtension = $this->container->get('bamboo_twig_loader.twig.render');
+  }
+
+  /**
+   * @covers ::renderBlock
+   *
+   * Cover the usage of
+   * {{ bamboo_render_block('block_content:ca1f2401-16a3-474b') }}.
+   * {{ bamboo_render_block('block_content:ca1f2401-16a3-474b', [], FALSE) }}.
```
Comment (WengerK) [inline tests/src/Kernel/Render/ContentBlockTest.php]:
```suggestion
   * {{ bamboo_render_block('block_content:ca1f2401-16a3-474b', [], TRUE) }}.
```

**Pull request merged**